### PR TITLE
{bio}[gompi/2020a] BLAST+ v2.10.1

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node126.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/77c374478d26321a3e5c6ddb53c8cafc for a full test report.

File: easybuild/easyconfigs/b/BLAST+/BLAST+-2.10.0-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+homepage = 'https://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'gompi', 'version': '2020a'}
+# Disabled vectorize, as enabling it leads to a compiler error on
+# Intel Sky Lake architecture.
+toolchainopts = {'usempi': True, 'vectorize': False}
```
Comment (lexming) [inline easybuild/easyconfigs/b/BLAST+/BLAST+-2.10.0-gompi-2020a.eb:26]:
```suggestion
toolchainopts = {'usempi': True, 'pic': True}
```

File: easybuild/easyconfigs/b/BLAST+/BLAST+-2.10.0-gompi-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.8'),
+    ('PCRE', '8.44'),
+    ('Boost', '1.72.0'),
+    ('GMP', '6.2.0'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('LMDB', '0.9.24'),
+]
+
```
Comment (lexming) [inline easybuild/easyconfigs/b/BLAST+/BLAST+-2.10.0-gompi-2020a.eb:44]:
```suggestion

# Disable auto-vectorization for the API on CPUs with AVX512 (Intel Skylake and onwards)
# Compilation fails on src/algo/blast/api/prelim_stage.cpp
local_apimake = 'src/algo/blast/api/Makefile.xblast.lib'
preconfigopts = "sed -i 's/FAST_CXXFLAGS)/FAST_CXXFLAGS) -fno-tree-vectorize/g' %s &&" % local_apimake

```

**Comment by FokkeDijkstra** [comment]
I'll look into this and update the BLAST+ version and incorporate your suggestions.

**Comment by FokkeDijkstra** [comment]
Just checked BLAST+ v2.10.1 and the compiler error with AVX512 is still in there. Will now test with the suggested fix.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c156f41a792d4c65b5765ee2092cf3d0 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @FokkeDijkstra !

**Pull request merged**